iml-gui/crate/src/auth.rs

```diff
@@ -95,6 +88,19 @@ pub fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg, GMsg>)
                 }
             };
         }
+        Msg::LoggedIn => {
+            orders.skip();
+            orders.send_msg(Msg::Fetch);
+            orders.send_g_msg(GMsg::RouteChange(Route::Dashboard.into()));
+        }
+
+        Msg::Logout => {
+            orders.perform_g_cmd(
```

Comment:
I think we could make a call of `event_source::init(orders);` from here to reset the records, but I don't this it is needed yet.

Reply:
Why would this be necessary?

Reply:
For example, if we had more sophisticated access control, some _data_ (e. g. filesystems) could be made hidden to users, and we would need to reset local state. This is hypothetical, I don't think we will ever need this :)